.github/CONTRIBUTING.md

```diff
@@ -56,6 +56,30 @@ approved by the reviewer.
 
 Once the pull request is approved, a team member will take care of merging.
 
+## Contributing training scripts
+
+The KerasCV team is working to include a catalog of high-performing model training scripts for the models included in KerasCV.models and KerasCV.applications, and is welcoming contributions for these scripts. These training scripts serve as documentation of good training techniques and will be used to train weights that will be offered in KerasCV models through the package.
+
+The KerasCV team will run submitted training scripts to produce weights for KerasCV, and will attribute strong weights to contributors via a training script ranking system. Stay tuned for more details about that.
+
+To contribute a script, start by opening an issue and tagging @ianjjohnson and @LukeWood to discuss the task, dataset, and/or model for which you'd like to add a script. Once they've taken a look, you can prepare a PR to introduce the new training script.
+
+See [this example PR](https://github.com/keras-team/keras-cv/pull/618) which added an initial training script for ImageNet classification. Please follow the structure of this training script in contributing your own script. New scripts should either:
+- Train a task for which we don't have a training script already
+- Include a meaningfully different training approach for a given task (e.g. a specific type of pretraining)
+- Introduce a custom training method for a specific model or dataset, based on empirical evidence of efficacy.
+
+Incremental improvements to existing training scripts are welcome, provided that they come with evidence of improved validation performance.
```

Comment:
lets recommend this before mentioning new scripts, as it is the primary contribution we expect